docs/howto/use-extensions/use-an-extension.rst

```diff
@@ -0,0 +1,15 @@
+.. _use-an-extension:
+
+Use an extension
+================
+
+To use an extension in an app, list it in the app's ``extensions`` key in the snap
+project file. Here's an example of an app using the KDE Neon 5 extension:
+
+.. code:: yaml
+
+    apps:
+      kcalc:
+        command: usr/bin/kcalc
+        extensions:
+          - kde-neon
```

Comment:
IDK whether this is relevant but we probably should use the neon 6 extension here if this file gets merged.

Reply:
Good catch.

Weirdly, this page was left out of #5212, which is the progenitor of these pages. Added back in, with the correction: https://github.com/canonical/snapcraft/pull/5212/commits/b993974f54764ba53442ff263128802d77d3330a